x-pack/plugins/cases/public/components/actions/assignees/edit_assignees_flyout.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import type { AppMockRenderer } from '../../../common/mock';
+import { createAppMockRenderer } from '../../../common/mock';
+import { basicCase } from '../../../containers/mock';
+import { waitForComponentToUpdate } from '../../../common/test_utils';
+import { EditAssigneesFlyout } from './edit_assignees_flyout';
+import { waitFor } from '@testing-library/react';
+
+jest.mock('../../../containers/user_profiles/api');
+
+describe('EditAssigneesFlyout', () => {
+  let appMock: AppMockRenderer;
+
+  /**
+   * Case one has the following assignees: coke, pepsi, one
+   * Case two has the following assignees: one, three
+   * All available assignees are: one, two, three, coke, pepsi
+   */
+  const props = {
+    selectedCases: [basicCase],
+    onClose: jest.fn(),
+    onSaveAssignees: jest.fn(),
+  };
+
+  beforeEach(() => {
+    appMock = createAppMockRenderer();
+    jest.clearAllMocks();
+  });
+
+  it('renders correctly', async () => {
+    const result = appMock.render(<EditAssigneesFlyout {...props} />);
+
+    expect(result.getByTestId('cases-edit-assignees-flyout')).toBeInTheDocument();
+    expect(result.getByTestId('cases-edit-assignees-flyout-title')).toBeInTheDocument();
+    expect(result.getByTestId('cases-edit-assignees-flyout-cancel')).toBeInTheDocument();
+    expect(result.getByTestId('cases-edit-assignees-flyout-submit')).toBeInTheDocument();
+
+    await waitForComponentToUpdate();
```

Comment:
Why use this at the end if you don't do any `expect` afterward? Even if something changes we won't notice.